webbeans-tomcat/src/site/site.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
```

Comment:
we don't really use maven site, maybe add some doc for http://svn.apache.org/repos/asf/openwebbeans/cms-site/trunk/content/ ?